.github/CODEOWNERS

```diff
@@ -445,6 +445,12 @@
 # ServiceLabel: %DataBox Edge
 # ServiceOwners:                                                   @a-t-mason @ganzee @manuaery
 
+# PRLabel: %Datadog
+/sdk/datadog/Azure.ResourceManager.*/                              @jbhanuchand
```

Comment:
```suggestion
/sdk/datadog/Azure.ResourceManager.*/                              @jbhanuchand @ArcturusZhang @ArthurMa1978
```

Reply:
This ensures that there are reviewers authorized to approve your PRs.   Generally, we ask that you include at least 2 people from your team.